ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -335,6 +340,17 @@ public void init() {
         (req, res) -> {
           String body = util.safeGetBody(req);
           List<MetacardChanges> metacardChanges = GSON.fromJson(body, METACARD_CHANGES_LIST_TYPE);
+          metacardChanges.forEach(
+              change -> {
+                EventType type = change.getType();
+                if (type != null) {
+                  EventApplication.notifyListeners(type);
+                } else {
+                  String unknownType =
+                      new Gson().fromJson(body, JsonObject.class).get("type").getAsString();
```

Comment:
❓ If `body` is the same `List<MetacardChanges>` that we already parsed, it doesn't seem like there would be a `type` property since it is a list of `MetacardChanges`. 

If you want the unknown type to be the default you can always add logic in the `getType` method to return `unknown` if it is null. If you do set it to return unknown type by default, then this whole block  could just become `change -> EventApplication.notifyListeners(change.getType())`